cluster-proxy/src/main/java/com/sequenceiq/cloudbreak/clusterproxy/ClusterProxyException.java

```diff
@@ -1,11 +1,22 @@
 package com.sequenceiq.cloudbreak.clusterproxy;
 
+import java.util.Optional;
+
 public class ClusterProxyException extends RuntimeException {
-    public ClusterProxyException(String message) {
-        super(message);
+
+    private final Optional<ClusterProxyError> clusterProxyError;
+
+    public ClusterProxyException(ClusterProxyError clusterProxyError) {
```

Comment:
when `ClusterProxyException` construction happens `ClusterProxyError` is already an optional. Shall this constructor accept the optional form so the rewrapping won't be necessary?